libs/utils/decorate.js

```diff
@@ -260,3 +262,25 @@ export function applyInViewPortPlay(video) {
     observer.observe(video);
   }
 }
+
+export function turnAnchorIntoVideo({ hash, src, anchorTag }) {
+  const { dataset, parentElement } = anchorTag;
+  const attrs = getVideoAttrs(hash, dataset);
+  const video = `<video ${attrs}></video>`;
+  anchorTag.insertAdjacentHTML('afterend', video);
+  createIntersectionObserver({
+    el: parentElement,
+    options: { rootMargin: `${ROOT_MARGIN}px` },
+    callback: () => {
+      parentElement
+        .querySelector('video')
+        .appendChild(
```

Comment:
I would use `?` for chaining.

Reply:
Hmm we do set the video element ourselves, so it should always exist. However given this is within an intersection observer, an outside function might manipulate the DOM and we'd have errors.

Added 👍